internal/update/update.go

```diff
@@ -20,7 +22,13 @@ type doer interface {
 // This is accomplished by fetching the latest github tag and comparing it to the version provided.
 // Returns the latest version, or an empty string if we're already running the latest version.
 // Will return ErrDevVersion if the build.Version is currently set to "dev".
-func Check(ctx context.Context, doer doer, version string) (string, error) {
+func Check() (string, error) {
```

Comment:
This should still accept an external `context.Context`.  The reason being we want to be able to cancel all the contexts by canceling the one root context.

Reply:
By the time you cancel the parent context in the CLI, the code has called `os.Exit` already. 

But, I could still pipe it through if you think it's important.

Reply:
I think it's still worth doing.